pyuvsim/version.py

```diff
@@ -76,7 +76,6 @@ def unicode_to_str(u):
 
 
 def main():
-    print("Rank = {0}".format(rank))
```

Comment:
@aelanman I took this line out to make the tests pass, is it needed for some reason?

Reply:
It should be fine. I think I just put that there to track what each rank was printing.